blocks/carousel/carousel.css

```diff
@@ -0,0 +1,490 @@
+    }
+}
+
+.explore-section .carousel-title {
+    flex: 0 0 33.3333%;
+    max-width: 33.3333%;
+}
+
+@media (width>= 768px) {
+    .explore-section .carousel-title {
+        flex: 0 0 50%;
+        max-width: 50%;
+    }
+}
+
+.explore-section .dropdowns {
+    text-align: right;
+    flex: 0 0 66.6666%;
+    max-width: 66.6666%;
+    position: relative;
+    width: 100%;
+    padding-right: 0.9rem;
+    padding-left: 0.9rem;
+    box-sizing: border-box;
+    outline: none;
+}
+
+@media (width>= 768px) {
+    .explore-section .dropdowns {
+        flex: 0 0 50%;
+        max-width: 50%;
+    }
+}
+
+.explore-section .dropdowns .dropdown-select {
+    margin-left: 0.6rem;
+    background-color: #fff;
+    display: inline-block;
+    margin-bottom: 0.3rem;
+    position: relative;
+}
+
+.explore-section .dropdowns .icon-down-arrow {
+    right: 0.6rem;
+    position: absolute;
+
+}
+
+.explore-section .dropdowns .icon-down-arrow::before {
+    color: #E87817;
+    content: "\e905";
+    font-size: 0.8rem;
+    font-weight: 800;
+    font-family: icomoon, sans-serif;
+
+}
+
+.explore-section .dropdowns .dropdown-toggle {
+    height: 1.9rem;
+    background-color: transparent;
+    font-size: 0.8rem;
+    text-transform: uppercase;
+    padding: 0 1.9rem 0 0.6rem;
+    border: 0.06rem solid #dee2e6;
+    border-radius: 0;
+    cursor: pointer;
+}
+
+.explore-section .dropdowns .dropdown-toggle .dropdown-text {
+    display: inline-block;
+    overflow: hidden;
+}
+
+.explore-section .dropdowns .dropdown-menu-container {
+    position: absolute;
+    inset: 0 auto auto 0;
+    z-index: 1000;
+    overflow: hidden;
+    will-change: transform;
+    transform: translate3d(0, 1.9rem, 0);
+    display: none;
+    float: left;
+    min-width: 100%;
+    padding: 0.5rem 0;
+    margin: 0.125rem 0 0;
+    font-size: 1rem;
+    color: #212529;
+    text-align: left;
+    list-style: none;
+    background-color: #fff;
+    background-clip: padding-box;
+}
+
+.explore-section .dropdowns .dropdown-menu {
+    float: none;
+    border: 0;
+    padding: 0;
+    margin: 0;
+}
+
+.explore-section .dropdowns .dropdown-menu  li {
+    font-size: 0.8rem;
+    text-transform: capitalize;
+    display: block;
+}
+
+.explore-section .dropdowns .dropdown-menu  li a {
+    padding: 0.3rem 0.6rem;
+    color: #415270;
+    cursor: pointer;
+    text-decoration: none;
+}
+
+
+.explore-section .carousel {
+    margin-bottom: 1.9rem;
+}
+
+.explore-section .carousel .value-col {
+    max-width: 50%;
+    margin: 0.8rem 0 0;
+}
+
+.explore-section .discover-more {
+    text-align: center;
+}
+
+@media (width >= 768px) {
+    .explore-section .discover-more {
+        text-align: right;
+    }
+
+}
+
+.explore-section .discover-more .link-color {
+    font-size: 1rem;
+}
+
+
+.explore-section .discover-more .link-color .icon-up-arrow {
+    transform: rotate(90deg);
+    margin-left: 0.3rem;
+}
+
+.explore-section.discover-more .link-color .icon-up-arrow::before {
```

Comment:
Instead of 'explore' or 'disocov?er' or 'research', should we have class names (also variables names) in the context of Carousel, so that same Carousel component can be used in future in other section of ICICI

Reply:
@shrotia updated.